src/main/kotlin/no/nav/yrkesskade/saksbehandling/service/BehandlingService.kt

```diff
@@ -396,4 +394,17 @@ class BehandlingService(
         )
     }
 
+    private fun hentDokumenterFraJournalpostResultat(journalpostResult : com.expediagroup.graphql.generated.Journalpost.Result?): List<DokumentInfo> {
+        return if (journalpostResult?.journalpost?.dokumenter != null) {
+            val journalpost = journalpostResult.journalpost
+            journalpost.dokumenter!!.map {
+                val dokument = it!!
+                val journalstatus = if (journalpost.journalstatus != null) journalpost.journalstatus.name  else "Status ikke satt"
+                val journalposttype = if (journalpost.journalposttype != null) journalpost.journalposttype.name else "Type ikke satt"
```

Comment:
Bare et forslag for å gjøre det litt penere her
```suggestion
                val journalstatus = journalpost.journalstatus?.name ?: "Status ikke satt"
                val journalposttype = journalpost.journalposttype?.name ?: "Type ikke satt"
```